vignettes/articles/05-read-the-outputs.Rmd

```diff
@@ -241,10 +279,66 @@ used.
 or bank for which the analysis was run.
 * `ald_sector`: Sector of the production assets analyzed.
 * `term`: Years to maturity of the loans.
-* `PD_change`: Change in probability of default of the the companies in the
-portfolio or loan book, aggregated to the sector level. This PD change shows
-variation of the impact on PDs by different maturities on the sector level.
+* `pd_change_shock`: Change in probability of default of the the companies in
+the portfolio or loan book, aggregated to the sector level. This PD change shows
+the variation of the impact on PDs by different maturities on the sector level.
 
 
 ### company_trajectories_\<suffix\>
-To be continued.
+
+This file contains an in detail overview of cash flow projections on the 
+company-technology level for the baseline and the shock scenarios.
+
+* `investor_name`: Name of the investor or bank for which the analysis was run.
+* `portfolio_name`: Name of the portfolio or loan book managed by the investor
+or bank for which the analysis was run.
+* `scenario_name`: Name that highlights the use of the Carbon budget.
+* `id`: internal id related to the company in the loan book or portfolio
```

Comment:
as id means something else for every asset type (and is NA for loans) I would encourage removing the columns

Reply:
will do for now although it could be helpful to assess coverage. But that is something we can think about at another point